### DISCUSSION TITLE
LAB2 co-sim under ubuntu 問題

### DISCUSSION THREAD

**Discussion by Spicychii** [bol-edu/HLS-SOC-Discussions#77] [Q&A] [answered]
錯誤訊息如附圖，正在進行的是FIRN11MAXI的co-sim，
請問這個錯誤是因為ubuntu下的讀檔位置不一樣造成的嗎？(因為console print out not found)
如果修改FIRTester.cpp的code第49行能解決這個情況的話，可以進行更改嗎？
註：另外於下方附上directives資訊，和版上大大對比過，應該是沒問題的(?
![螢幕擷取畫面 2023-09-29 004458](https://github.com/bol-edu/HLS-SOC-Discussions/assets/45990396/2fcf4742-0931-48c9-92e3-75db33830d70)
![螢幕擷取畫面 2023-09-29 005234](https://github.com/bol-edu/HLS-SOC-Discussions/assets/45990396/e81d512a-176b-4cfd-9706-c90ce0e883b0)

**Comment by holyuming** [comment, marked as answer]
他顯示sh fc not found

是因為Ubuntu Linux並沒有fc的command，需改為diff，且斜線部分也要改成Linux可讀的方式，可看前幾篇的討論
